upload.py

```diff
@@ -14,11 +14,14 @@ def main():
         help="package type",
     )
     args = parser.parse_args()
+
     tag = f"dvc-{args.pkg}"
-    image = {"deb": "ubuntu", "rpm": "fedora"}[args.pkg]
-    docker_dir = f"docker/{image}"
-    target = "upload" if image == "ubuntu" else None
+    image = {"deb": "ubuntu", "rpm": "centos"}[args.pkg]
```

Comment:
I'm assuming  that `upload.sh` will also work on `centos`, as long as `gnupg` and `rpm-sign` are also installed